src/SiteLink.php

```diff
@@ -18,10 +18,18 @@
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  * @author Michał Łazowik
+ * @author Thiemo Mättig
```

Comment:
I did look at the git blame and did not see much done by you. I checked again now and getting a different impression. Perhaps I incorrectly checked ItemIdSet. My bad, I don't object to you adding your name here.

Can you please try to assume good faith and not throw accusations at me?